@here/harp-examples/src/datasource_satellitetile_globe.ts

```diff
@@ -34,7 +34,7 @@ export namespace SatelliteTileDataSourceGlobeExample {
 
         const map = new MapView({
             canvas,
-            theme: "resources/berlin_tilezen_base.json",
+            theme: "resources/berlin_tilezen_base_globe.json",
```

Comment:
Is there no way to use the existing style? Duplicating them for globe sounds like an overkill and a big maintenance overhead

Reply:
It's true that for now they only differ in the ``Sky`` property, but decided to added because in the future designer might want to tweak other properties (i.e. text size).

Reply:
@musculman: This is what I mentioned in the standup as well. We should support json $ref so we could reuse parts of existing stylesheets. @robertoraggi mentioned that this is also covered by his style refactoring.